signup/tool/src/java/org/sakaiproject/signup/tool/util/Utilities.java

```diff
@@ -376,15 +376,16 @@ public static List<SelectItem> getMeetingTypeSelectItems(
 	 *            a detailed action info string
 	 */
 	public static void postEventTracking(String mainSignupEventType,
-			String eventActionInfo) {
+			String placementContext, Long meetingId, String meetingTitle, String eventActionInfo) {
 		if (postToDatabase) {
+			String eventContent = placementContext + " meetingId|title:" + meetingId + "|" + meetingTitle + " " + eventActionInfo;
```

Comment:
Hardcoded text?

```
meetingId|title:
```

Reply:
This hardcoded text was essentially already present in other places, I just moved it here for consistency